oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/importer/IndexImporter.java

```diff
@@ -69,6 +74,7 @@
     private final IndexEditorProvider indexEditorProvider;
     private final AsyncIndexerLock indexerLock;
     private final IndexDefinitionUpdater indexDefinitionUpdater;
+    private final boolean preserveCheckpoint = Boolean.parseBoolean(System.getProperty(OAK_INDEX_IMPORTER_PRESERVE_CHECKPOINT, "false"));
```

Comment:
I think it is equivalent to Boolean.getBoolean(x): https://docs.oracle.com/javase/9/docs/api/java/lang/Boolean.html#getBoolean-java.lang.String-

Reply:
done.